crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -318,6 +328,10 @@ fn build_new_replicated_loglet_configuration(
         })
         .collect();
 
+    let sequencer = preferred_sequencer
+        .and_then(|node_id| observed_cluster_state.alive_nodes.get(&node_id))
+        .or_else(|| log_servers.iter().choose(&mut rng).cloned());
+
```

Comment:
If we accept NodeId as input, we can fallback to the latest generation of the same node if the generation is too old maybe? albeit, it wouldn't matter much and we can choose any other alive node as you're proposing as well.

Does `log_servers` include nodes with Role log-server only? The sequencer can run on any node in the cluster, not only on log-servers.